packages/php/blueprint/src/Exporters/ExportInstallThemeSteps.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Automattic\WooCommerce\Blueprint\Exporters;
+
+use Automattic\WooCommerce\Blueprint\Steps\InstallTheme;
+use Automattic\WooCommerce\Blueprint\UseWPFunctions;
+
+class ExportInstallThemeSteps implements StepExporter {
+	use UseWPFunctions;
+
+	public function export() {
+		$steps        = array();
+		$thmes        = $this->wp_get_themes();
+		$active_theme = $this->wp_get_theme();
+
+		foreach ( $thmes as $slug => $theme ) {
+			// Check if the theme is active
+			$is_active = $theme->get( 'Name' ) == $active_theme->get( 'Name' );
```

Comment:
Should we use type sensitive check here?
```suggestion
			$is_active = $theme->get( 'Name' ) === $active_theme->get( 'Name' );
```

Reply:
How did I miss it 🤦 I thought I already have muscle memory to use === all the time by now. Thank you!